The job **`dist-x86_64-apple`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/6842425282/job/18603883436) [(plain)](https://github.com/rust-lang-ci/rust/commit/9f99edcef63e3d1336c69e21a27f74e3743f12f9/checks/18603883436/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
SCRIPT=./x.py dist bootstrap --include-default-paths --host=x86_64-apple-darwin --target=x86_64-apple-darwin
SELECT_XCODE=/Applications/Xcode_13.4.1.app
SHELL=/bin/bash
SHLVL=3
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.Vnhgfj1CUy/Listeners
STATS_EXT=true
STATS_EXTP=https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data
STATS_RDCL=true
STATS_TIS=mining
---
pkgbuild: Wrote package to /Users/runner/work/rust/rust/build/tmp/dist/combined-tarball/pkg/rust-docs.pkg
pkgbuild: Adding top-level postinstall script
pkgbuild: Wrote package to /Users/runner/work/rust/rust/build/tmp/dist/combined-tarball/pkg/rust-demangler.pkg
thread 'main' panicked at src/lib.rs:1731:25:
could not read dir "/Users/runner/work/rust/rust/build/tmp/tarball/rust/x86_64-apple-darwin/rustc-codegen-cranelift-beta-x86_64-apple-darwin": Os { code: 2, kind: NotFound, message: "No such file or directory" }
Build completed unsuccessfully in 1:26:50
  local time: Sun Nov 12 20:10:17 UTC 2023
  network time: Sun, 12 Nov 2023 20:09:19 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/src/bootstrap/src/core/build_steps/dist.rs
+++ b/src/bootstrap/src/core/build_steps/dist.rs
@@ -1630,13 +1630,10 @@ impl Step for Extended {
             prepare("rust-analysis");
             prepare("clippy");
             prepare("rust-analyzer");
-            for tool in &["rust-docs", "rust-demangler", "miri"] {
+            for tool in &["rust-docs", "rust-demangler", "miri", "rustc-codegen-cranelift"] {
                 if built_tools.contains(tool) {
                     prepare(tool);
                 }
-            }
-            if builder.config.rust_codegen_backends.contains(&INTERNER.intern_str("cranelift")) {
-                prepare("rustc-codegen-cranelift");
             }
             // create an 'uninstall' package
             builder.install(&etc.join("pkg/postinstall"), &pkg.join("uninstall"), 0o755);

